kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-rootless.yaml

```diff
@@ -0,0 +1,38 @@
+- name: ci-kubernetes-e2e-kubeadm-kinder-rootless-{{ dashVer .KubernetesVersion }}
```

Comment:
this looks fine to me.
could you please send the kubernetes/test-infra PR as well?

running ./hack/update-workflows.sh would write the new file in the test-infra clone that you have.

Reply:
Ill do that once we merge this to avoid having to keep both in sync.